modin/experimental/core/execution/native/implementations/hdk_on_native/df_algebra.py

```diff
@@ -1036,7 +1038,7 @@ class SortNode(DFAlgNode):
         Holds a single sorted frame.
     columns : list of str
         A list of key columns for a sort.
-    ascending : bool
+    ascending : list of bool
```

Comment:
list with one element?

Reply:
In my understanding, it means a list containing elements of type bool. Should it be `list of bools`?

Reply:
no, I just missed that this parameter can be a list. Do you have a test for this case?

Reply:
Ah, sorry, didn't get the question. It should always be a list. No, there is no a special test case.